.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
Can you remove this file? Maybe it should be gitignored?

Reply:
Yes. But I think it's useful?

Reply:
It is useful, but we generally just use [standard](http://ghub.io/standard) as our opinionated linter of choice on Electron projects.

Reply:
I don't see that in dev mode? Can we add `eslint` for that word on local?

Reply:
npm test: `NODE_ENV=test mocha && standard --fix`

Reply:
Thanks! I will update this PR's branch to update this scripts.